app/src/main/java/ch/epfl/sdp/drone3d/service/impl/drone/DroneExecutorImpl.kt

```diff
@@ -45,21 +45,24 @@ class DroneExecutorImpl(
 
         val instance = getInstance()
 
```

Comment:
Is this file modified for another reason than formatting ?
Because it dosen't seem too and this will create merging confilts with further commits. I would prefere it to be reverersed

Reply:
No it is just automatic reformatting done by Android Studio when I was doing some tests. I just reversed it.